content/posts/2023/reana-0.9.1.md

```diff
@@ -64,7 +64,7 @@ To learn more about this feature, please have a look at the related [Configuring
 
 There are many improvements in stability and performance of the REANA cluster that are coming with the 0.9.1 release update:
 
-- The periodic quota updater for disk and CPU quotas is now up to 70% faster.
+- The periodic quota updater for disk and CPU quotas is now up to 3.4 times faster.
```

Comment:
I'll change the text to _"up to four times"_ for indication, because 3.4 seems to be a too precise number in the phrase... I guess the exact number may vary from installation to installation, based on hardware, disk systems, number of users and workflows, etc.